src/work-allocation-2/containers/task-list/task-list.component.html

```diff
@@ -1,101 +1,98 @@
-<div class="task-list-container">
-  <div *ngIf="pagination; else tableSummary" class="pagination-top" attr.aria-label="{{ tasksTotal }} results have been found" role="status">
-    <span class="text-16" id="search-result-summary__text">Displaying
-        <span class="govuk-!-font-weight-bold">{{ getFirstResult() }}</span>
-        -
-        <span class="govuk-!-font-weight-bold">{{ getLastResult() }}</span>
-        out of
-        <span class="govuk-!-font-weight-bold">{{ tasksTotal || 0 }}</span>
-        tasks.</span>
-  </div>
+<div *ngIf="pagination; else tableSummary" class="pagination-top" attr.aria-label="{{ tasksTotal }} results have been found" role="status">
+  <span class="text-16" id="search-result-summary__text">Displaying
+      <span class="govuk-!-font-weight-bold">{{ getFirstResult() }}</span>
+      -
+      <span class="govuk-!-font-weight-bold">{{ getLastResult() }}</span>
+      out of
+      <span class="govuk-!-font-weight-bold">{{ tasksTotal || 0 }}</span>
+      tasks.</span>
+</div>
 
-  <ng-template #tableSummary>
-    <div class="pagination-top" attr.aria-label="{{ tasksTotal }} results have been found" role="status">
-      <span class="text-16" id="search-result-summary__text">Displaying <span class="govuk-!-font-weight-bold">{{ tasksTotal || 0 }} </span>tasks.</span></div>
-  </ng-template>
+<ng-template #tableSummary>
+  <div class="pagination-top" attr.aria-label="{{ tasksTotal }} results have been found" role="status">
+    <span class="text-16" id="search-result-summary__text">Displaying <span class="govuk-!-font-weight-bold">{{ tasksTotal || 0 }} </span>tasks.</span></div>
```

Comment:
_Nitpick_:
Maybe wrap the closing `</div>` onto the next line?

Reply:
I assume you meant line 21? Changed.

Reply:
No, the one at the end of line 13. It's the closing tag for the `<div>` on line 12.

Reply:
I've formatted the entire file now.